lua/typst-preview/utils.lua

```diff
@@ -53,7 +53,12 @@ end
 ---Open link in browser (platform agnostic)
 ---@param link string
 function M.visit(link)
-  local cmd = string.format('%s http://%s', open_cmd, link)
+  local cmd
+  if config.opts.open_cmd ~= '' then
+    cmd = string.format(config.opts.open_cmd, link)
```

Comment:
Can you prepend `'http://'` to `link` so that people don't have to include `'http://'` in their `open_cmd`. In my opinion it would also make more sense.